### PR TITLE
Rel 0.1.5 rapids

### PR DESCRIPTION
**Pull request by NvTimLiu**
Updated to the latest release of version "beta5"

### CODE REVIEW

File: examples/notebooks/scala/mortgage_gpu_crossvalidation.ipynb [hunk 1]
```diff
@@ -0,0 +1,536 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "trainParquetPath = /home/bobwang/jupyter/data/mortgage/cv/train\n",
```
Comment (GaryShen2008) [inline examples/notebooks/scala/mortgage_gpu_crossvalidation.ipynb]: Let's correct this path as same as the path used in other sample code.

File: examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "trainParquetPath = /home/bobwang/jupyter/data/taxi/cv\n",
```
Comment (GaryShen2008) [inline examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb]: Let's use the same path in the other files.

File: getting-started-guides/csp/gcp/gcp.md [hunk 3]
```diff
@@ -200,7 +200,7 @@ Submitting a CPU job on this cluster is very similar. Below's an example command
 ```bash
     export GCS_BUCKET=my-bucket
     export MAIN_CLASS=ai.rapids.spark.examples.mortgage.CPUMain
-    export RAPIDS_JARS=gs://$GCS_BUCKET/spark-gpu/sample_xgboost_apps-0.1.4-jar-with-dependencies.jar
+    export RAPIDS_JARS=gs://$GCS_BUCKET/spark-gpu/sample_xgboost_apps-0.1.5-jar-with-dependencies.jar
     export DATA_PATH=gs://$GCS_BUCKET
     export TREE_METHOD=hist
     export SPARK_NUM_EXECUTORS=4
```
Comment (GaryShen2008) [inline getting-started-guides/csp/gcp/gcp.md]: Do we really need gcp/pics/gcp-bucket.png?

File: examples/notebooks/scala/mortgage_gpu_crossvalidation.ipynb (hunk 1 above)
Comment (NvTimLiu) [reply examples/notebooks/scala/mortgage_gpu_crossvalidation.ipynb]: /data/mortgage/parquet/xxx

File: examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb (hunk 2 above)
Comment (NvTimLiu) [reply examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb]: changed path to : /data/mortgage/parquet/xxx

File: getting-started-guides/csp/gcp/gcp.md (hunk 3 above)
Comment (NvTimLiu) [reply getting-started-guides/csp/gcp/gcp.md]: rm file : pics/gcp-bucket.png

**Pull request merged**